README.md

```diff
@@ -386,6 +386,19 @@ To benchmark evaluating a feature's variable via SDKs's `.getVariable()` method:
   -n 100
 ```
 
+### Evaluate
+To learn why certain values (like feature and its variation or variables) are evaluated as they are against provided [context](https://featurevisor.com/docs/sdks/javascript/#context):
+
+```bash
+ FeaturevisorTestRunner evaluate \
+  --environment staging \
+  --feature feature_key \
+  --feature variable_key \
```

Comment:
do we need this line?

Reply:
Nooooo, it's a copy & paste the issue. Removed